canonical/lxd-ui
# Rename header component and rename project

## Done

- factored out a common RenameHeader component
- added option to rename the project

Fixes [WD-3352](https://warthogs.atlassian.net/browse/WD-3352)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Check the headers for instances detail, profile detail and project configuration while or before and after renaming.

[WD-3352]: https://warthogs.atlassian.net/browse/WD-3352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
